service/setting_service.py

```diff
@@ -27,7 +27,7 @@ def get_settings():
 async def save_settings(request):
     data = await request.json()
     json_data = data
-    file_name = f'{db_dir_path}/settings.json'
+    file_name = f'{db_dir_path}/userSettings.json'
```

Comment:
Hi! thanks for looking into this. i'm actually trying to migrate to settings.json, because as you can see, the userSettings.json is stringified json and not very readable. settings.json is the real readable .json file that you can modify. 

But I just haven't implement the logic to migrate the old settings data into the new settings.json ...

Reply:
![Screenshot 2024-06-30 025258](https://github.com/11cafe/comfyui-workspace-manager/assets/13986467/4ed85b0a-d601-4dc9-871a-4a765d13f0b5)

Currently the function of changing "My Workflows Save Directory" is broken, once I click "Save" and reload, the save directory reverts back to the default one. Do you have any suggestions on how to make it works?